tests/sys/conn_can/Makefile.ci

```diff
@@ -1,25 +1,3 @@
 BOARD_INSUFFICIENT_MEMORY := \
-    airfy-beacon \
-    calliope-mini \
-    i-nucleo-lrwan1 \
-    im880b \
-    microbit \
-    nrf51dongle \
-    nucleo-f030r8 \
-    nucleo-f031k6 \
-    nucleo-f042k6 \
-    nucleo-f070rb \
-    nucleo-f072rb \
-    nucleo-f302r8 \
-    nucleo-f303k8 \
-    nucleo-f303re \
-    nucleo-f334r8 \
-    nucleo-l031k6 \
-    nucleo-l053r8 \
-    saml10-xpro \
-    saml11-xpro \
-    stm32f030f4-demo \
-    stm32f0discovery \
-    stm32l0538-disco \
-    yunjia-nrf51822 \
```

Comment:
Those likely still have insufficient memory

Reply:
But they don't have a CAN interface, so we won't build for them anyway :)